Makefile

```diff
@@ -3,7 +3,14 @@ version = 25
 srcdir = .
 VPATH = $(srcdir)
 
-PREFIX = /usr/local
+ifeq "$(PREFIX)" ""
+	INCLUDEPREFIX = /usr/include
+else
+	INCLUDEPREFIX := $(PREFIX)/include
+	PREFIX := $(PREFIX)/local
```

Comment:
So if I set `PREFIX` to `/usr/local`, the prefix used will be `/usr/local/local`? That's not how it's supposed to work...

Reply:
This piece of code presumes that `$PREFIX` isn't normally set on traditional filesystems, but it is set by default on Termux. If defining `$PREFIX` must be supported on normal filesystems, too, I might change the `if` test to something like

```
ifeq "$(PREFIX)" ""
	INCLUDEPREFIX = /usr/include
else ifeq "$(PREFIX)" "/data/data/com.termux/files/usr/"
	INCLUDEPREFIX := $(PREFIX)/include
	PREFIX := $(PREFIX)/local
endif
endif
```

I not tested this yet, I must check the syntax before.